openedx/core/djangoapps/theming/models.py

```diff
@@ -0,0 +1,39 @@
+"""
+Django models supporting the Comprehensive Theming subsystem
+"""
+from django.db import models
+from django.conf import settings
+from django.contrib.sites.models import Site
+
+
+class SiteTheme(models.Model):
```

Comment:
What is the motivation for having a `SiteTheme` model and not a `SiteConfiguration` model in which we could persist the `theme_dir_name` value as well as the other site configuration values that currently live in the MICROSITE_CONFIGURATION setting?